components/voilanorbert/actions/find-contact/find-contact.mjs

```diff
@@ -0,0 +1,32 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "voilanorbert-find-contact",
+  name: "Find Contact",
+  description: "This action returns a specific contact. The object email is either null when the email is not found, or contains an object with at least email (the email string) and the score. [See the docs here](https://api.voilanorbert.com/2018-01-08/#contacts-get-1)",
+  version: "0.0.1",
+  type: "action",
+  props: {
```

Comment:
Can we use [this one](https://api.voilanorbert.com/2018-01-08/#contacts-get) instead of [this](https://api.voilanorbert.com/2018-01-08/#contacts-get-1) and can we add other search parameters like `query`, `name`, `email`?

Reply:
I think that "find contact" is looking a specific contact, [this one](https://api.voilanorbert.com/2018-01-08/#contacts-get) is looking for more than one contact. I think they are different behaviors